examples-spark/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>onyx-project</artifactId>
+        <groupId>edu.snu.onyx</groupId>
+        <version>0.1-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+
+    <artifactId>onyx-examples-spark</artifactId>
+
+    <repositories>
+        <repository>
+            <id>Bundled Maven Repository</id>
+            <url>file://${basedir}/../common/src/main/resources/repository</url>
+        </repository>
+    </repositories>
+    <name>Onyx Examples: Spark</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>edu.snu.onyx</groupId>
+            <artifactId>onyx-compiler-frontend-spark</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.fommil.netlib</groupId>
+            <artifactId>all</artifactId>
+            <version>${netlib.version}</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-mapreduce-client-core</artifactId>
+            <version>${hadoop.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <version>${hadoop.version}</version>
+        </dependency>
+        <!--tests-->
```

Comment:
Can you remove dependencies to testing frameworks?

Reply:
That means moving everything into the test/ directory, which is undesirable. Can you elaborate?

Reply:
Ok. I see `examples-beam` also depend on the testing frameworks, presumably due to some dependency problems, which I guess are out of scope of this PR.

For this PR, maybe share code with `examples-beam/pom.xml`, if possible?

Reply:
I'll simply add the tests dependency to the project pom then.